app/auth/views/register.py

```diff
@@ -60,6 +61,7 @@ def register():
                     hcaptcha_res,
                 )
                 flash("Wrong Captcha", "error")
+                RegisterEvent(RegisterEvent.ActionType.regsiter_catpcha_failed).send()
```

Comment:
There seems to be a typo in `regsiter_catpcha_failed`

Reply:
Good catch :ok_hand: